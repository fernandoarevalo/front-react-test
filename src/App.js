import React from "react";
import "materialize-css/dist/css/materialize.min.css";
import M from "materialize-css";
import Form from "components/Form";
import Menu from "components/Menu";

function App() {
  M.AutoInit();
  return (
    <React.Fragment key="app">
      <header>
        <Menu />
      </header>
      <main>
        <Form />
      </main>
    </React.Fragment>
  );
}

export default App;
