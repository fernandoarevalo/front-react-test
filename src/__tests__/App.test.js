import React from "react";
import { render } from "@testing-library/react";
import App from "App";

describe("<App />", () => {
  test("Snapshot app component", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
