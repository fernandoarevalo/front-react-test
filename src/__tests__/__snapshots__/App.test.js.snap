// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshot app component 1`] = `
<DocumentFragment>
  <header>
    <ul
      class="sc-gsTCUz lgfLoM card collection"
      data-testid="menu-container"
    >
      <li
        aria-selected="true"
        class="collection-item active"
        role="tab"
      >
        <button
          class="sc-bdfBwQ hGbOiT"
          type="button"
        >
          Vivaair
        </button>
      </li>
      <li
        aria-selected="false"
        class="collection-item "
        role="tab"
      >
        <button
          class="sc-bdfBwQ hGbOiT"
          type="button"
        >
          Avianca
        </button>
      </li>
    </ul>
    <section
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col s12"
        >
          <h4>
            Hola, bienvenido, sabemos que quieres viajar en un Vivaair
          </h4>
        </div>
      </div>
    </section>
  </header>
  <main>
    <section
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col s12 l6 offset-l3"
        >
          <form
            class="card-panel"
          >
            <div
              class="input-field"
            >
              <input
                class="validate"
                id="name"
                name="name"
                type="text"
              />
              <label
                for="last_name"
              >
                Nombre
              </label>
            </div>
            <div
              class="input-field"
            >
              <input
                class="validate"
                id="email"
                name="email"
                type="email"
              />
              <label
                for="email"
              >
                Email
              </label>
            </div>
            <div
              class="input-field"
            >
              <input
                class="validate"
                id="cellphone"
                name="cellphone"
                type="text"
              />
              <label
                for="cellphone"
              >
                Celular
              </label>
            </div>
            <div
              class="input-field"
            >
              <input
                class="validate"
                id="age"
                name="age"
                type="number"
              />
              <label
                for="age"
              >
                Edad
              </label>
            </div>
            <button
              class="waves-effect waves-light btn"
              type="submit"
            >
              Enviar
            </button>
          </form>
        </div>
      </div>
    </section>
    <div
      class="modal"
      id="modal"
    >
      <div>
        <div
          class="modal-content"
        >
          <h4>
            Grandioso!
          </h4>
          <p>
            Tu información fue enviada con éxito, estaremos en contacto contigo
          </p>
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
